src/scopes.ts

```diff
@@ -668,6 +670,20 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
     type: ScopeType.Modify,
     products: [TranscendProduct.Assessments, TranscendProduct.DataMapping],
   },
+  [ScopeName.ViewAssignedAssessments]: {
+    title: 'View Assessments',
```

Comment:
```suggestion
    title: 'View Assigned Assessments',
```

Reply:
damn, sorry haha opening a new one. Thanks for catchin!